funnel/templates/rsvp.html

```diff
@@ -0,0 +1,2 @@
+{% from "macros.html" import rsvp with context %}
```

Comment:
Avoid importing `with context`. Pass the macro whatever it needs. Makes debugging easier if something changes in future.